TriforkSwiftExtensions/TriforkSwiftExtensions/Reusable/UITableViewCellExtension.swift

```diff
@@ -8,7 +8,9 @@
 
 import UIKit
 
-extension UITableViewCell: Reusable {
+extension UITableViewCell {
+
+    /// Id of cell. Used to register and deque
     public static var id: String {
```

Comment:
Hmn, should this be a function to indicate that it is an auto-generated id and not something Apple provides on the `UITableViewCell`?

Reply:
I think we can make this internal actually. As the dequeue part is only used in this framework 🤔

Reply:
or we should change it to `public static var reuseId: String`

Reply:
As far as I remember the `UITableViewCell` already has the `reuseIdentifier` property, so I think it is kind of confusing with two `var` properties that are almost the same. I'd prefer it as a function, because it generates an ID for the cell. But I think it can be useful as a `public` function too!
Also, the documentation of it should make clear how it generates the ID and how it is used internally in the framework (sorry I didn't catch this the first time I reviewed it 😅)

Reply:
True that @tkctrifork !

The `var reuseIdentifier: String?` is a non-static. But i agree on it can be confusing. 
I actually prefer a `static var` and not a function. 

I like to say that var is for simple implementation, with no side effect. 
Functions are for more heavy computation or have side effects. This is a hard balance 👀

I don't see a use case for making it public? It just returns a long strange string.

I will update the documentation. to generate it. 

No problem! We get better every day 💪

Reply:
Oh shit, I didn't see that it was `static` 💩
I can live with it! Sorry about the confusion. Let's make it `internal` and keep it `static var`? 💃